scripts/globals/abilities/super_jump.lua

```diff
@@ -22,4 +23,18 @@ end;
 -----------------------------------
 
 function onUseAbility(player,target,ability)
-end;
\ No newline at end of file
+
+    -- Reduce 99% of total accumulated enmity
+    if (target:isMob()) then
+        target:lowerEnmity(player, 99);
+    end
+    
+    ability:setMsg(0);
+    
+    -- If the Dragoon's wyvern is out and alive, tell it to use Super Climb
+    local wyvern = player:getPet();
+    if (wyvern ~= nil and player:getPetID() == PET_WYVERN and wyvern:getHP() > 0) then 
+        wyvern:useJobAbility(636, wyvern);
+    end
+
```

Comment:
iirc the Drg can't be targeted by pt members while in the air. I played a drg for many years on retail, but still I'm going off of memory.

Ffxiclopedia has some decent info in the notes section:
http://ffxiclopedia.wikia.com/wiki/Super_Jump